action.yml

```diff
@@ -22,3 +22,5 @@ outputs:
 runs:
   using: 'node16'
   main: 'lib/action.js'

```

Comment:
If the customer only wants container-mapping stuff, the action.js will still run the other tools right? Is there a way to avoid those tools? 
Additionally, if the customer doesnt want container-mapping, I think we shouldn't push the data out of the MSDO client. We should add a check for that too.

Reply:
Ported over the `commands`. PreJob and PostJob will be no-ops if you explicitly specify 'pre' or 'post', instead running as "main" which is where the task is located.  I'm looking into if we can do pre-if and post-if based on input commands.